components/base/__tests__/FilterTabs.test.tsx

```diff
@@ -1,18 +1,17 @@
 import renderer from "react-test-renderer";
-import ReactTestUtils from "react-dom/test-utils";
-import FilterTabs from "../FilterTabs";
+import FilterTabs, { TabStyle } from "../FilterTabs";
 
-test("Filter Tabs", () => {
+const TABS = ["GET BUILDING", "PREREQUISITES", "FEATURES"];
+
+test("FilterTabs", () => {
   const setState = jest.fn();
-  const tree = renderer
-    .create(
-      <FilterTabs
-        tabs={["GET BUILDING", "PREREQUISITES", "FEATURES"]}
-        filter={null}
-        setFilter={setState}
-      />
-    )
-    .toJSON();
-  ReactTestUtils.Simulate.click(tree);
-  expect(tree).toMatchSnapshot();
+  const tree = renderer.create(
+    <FilterTabs tabs={TABS} filter={null} setFilter={setState} />
+  );
+  tree.root.findAllByType(TabStyle).map((t) => t.props.onClick());
+
+  expect(setState.mock.calls.map((c) => c)).toEqual(
```

Comment:
`setState.mock.calls` is an array of arguments the mock function has been called with. In this case it should equal:

```javascript
[
  [null],
  ["GET BUILDING"],
  ["PREREQUISITES"],
  ["FEATURES"]
]
```